mock-relying-party-ui/Dockerfile

```diff
@@ -15,7 +20,11 @@ ENV REDIRECT_URI_REGISTRATION=$redirect_uri_registration
 ENV CLIENT_ID=$client_id
 ENV ACRS=$acrs
 ENV SIGN_IN_BUTTON_PLUGIN_URL=$sign_in_button_plugin_url
-
+ENV DISPLAY=$display
+ENV PROMPT=$prompt
+ENV GRANT_TYPE=$grant_type
+ENV MAX_AGE=$max_age
+ENV CLAIMS_LOCALES=$claims_locales
```

Comment:
these values should be added on line 99, similar to ESIGNET_UI_BASE_URL or MOCK_RELYING_PARTY_SERVER_URL